modules/arm_plugin/src/arm_converter/arm_converter_gather.cpp

```diff
@@ -46,6 +46,24 @@ template <> Converter::Conversion::Ptr Converter::Convert(const ngraph::op::v1::
         node.input(1), indexTypes);
 }
 
+template <> Converter::Conversion::Ptr Converter::Convert(const ngraph::op::v8::Gather& node) {
```

Comment:
I think this implementation is unnecessary. It's not registered and anyway won't be used  since transformation replace all Gv8 nodes with Gv7

Reply:
@terfendail done, thank you